Twister "FLASH overflow" does not account for imgtool trailer.

**Describe the bug**
Twister has a check for skipping (rather than failing) builds that trigger a flash overflow error. That does not work for board that call imgtool through, where the build image is under the flash size, but the image + imgtool trailer is not. The build fails CI with an error like:

```
...
Memory region         Used Size  Region Size  %age Used
           FLASH:       97780 B        96 KB     99.47%
            SRAM:       27544 B       136 KB     19.78%
        USB_SRAM:          0 GB        16 KB      0.00%
        IDT_LIST:          0 GB         2 KB      0.00%
Usage: wrapper.py [OPTIONS] INFILE OUTFILE
Try 'wrapper.py --help' for help.

Error: Image size (0x1819e) + trailer (0x1b0) exceeds requested size 0x18000
sign the payload
ninja: build stopped: subcommand failed.
```

**To Reproduce**
Steps to reproduce the behavior:
1. git checkout v3.2.0-rc2
2. west build -p -b lpcxpresso55s69_ns samples/net/sockets/http_get

**Expected behavior**
The test should be skipped.

**Impact**
Annoyance, unrelated CI failures.

**Logs and console output**
n/a

**Environment (please complete the following information):**
 - OS: Linux
 - Toolchain: Zephyr SDK 0.15.0
 - Commit SHA: v3.2.0-rc2

--- a/scripts/pylib/twister/twisterlib/runner.py
+++ b/scripts/pylib/twister/twisterlib/runner.py
@@ -220,10 +220,14 @@ class CMake:
 
             if log_msg:
                 overflow_found = re.findall("region `(FLASH|ROM|RAM|ICCM|DCCM|SRAM|dram0_1_seg)' overflowed by", log_msg)
+                imgtool_overflow_found = re.findall(r"Error: Image size \(.*\) \+ trailer \(.*\) exceeds requested size", log_msg)
                 if overflow_found and not self.options.overflow_as_errors:
                     logger.debug("Test skipped due to {} Overflow".format(overflow_found[0]))
                     self.instance.status = "skipped"
                     self.instance.reason = "{} overflow".format(overflow_found[0])
+                elif imgtool_overflow_found and not self.options.overflow_as_errors:
+                    self.instance.status = "skipped"
+                    self.instance.reason = "imgtool overflow"
                 else:
                     self.instance.status = "error"
                     self.instance.reason = "Build failure"

